Manuscript/Moreno_and_Sasaki_2023.Rmd

```diff
@@ -37,11 +37,11 @@ knitr::opts_chunk$set(fig.align = "center")
 
-These interactions, however, between starvation and thermal limits have been scantly studied in marine organisms. [NEED MORE HERE - quick summary of how starvation affects CTmax, pulling from both marine and non-marine taxa would be great]. Beyond the direct effects of starvation on thermal limits, @huey2019 describe a process termed 'metabolic meltdown', in which exposure to high temperatures, reduced food intake, and decreasing thermal limits act synergistically to drastically decrease tolerance to a warmer environment. A key component of metabolic meltdown is the reduction of thermal limits under reduced food intake. In order to better understand the vulnerability of marine systems to increasing temperatures, it is vital to determine how starvation affects thermal limits.
+These interactions, however, between starvation and thermal limits have been scantly studied, especially in aquatic ectotherms. Most experiments which examine this relationship have focused on terrestrial taxa, including Diptera [@manenti2018; @gotcha2018; @mitchell2017], Coleoptera [@chidawanyika2017; @scharf2016], Hymenoptera [@nguyen2017, @gonzalez2022], Hempitera [@devries2016], and Lepidotera [@mutamiswa2018, @mir2018].These studies vary in methodology and results, but generally indicate that organismal responses to starvation and heat tolerance are closely related. Moreover, Experiments on the freshwater amphipod *Gammarus fossarum* revealed that starvation improved survival during acute heat stress, but had no effect on mild heat stress [@semsar-kazerouni2020]. Beyond the direct effects of starvation on thermal limits, @huey2019 describes a process termed 'metabolic meltdown', in which exposure to high temperatures, reduced food intake, and decreasing thermal limits act synergistically to drastically decrease tolerance to a warmer environment. A key component of metabolic meltdown is the reduction of thermal limits under reduced food intake. In order to better understand the vulnerability of marine systems to increasing temperatures, it is vital to determine how starvation affects thermal limits.
```

Comment:
Nicely done with the summary! I made some additional minor edits to the text, if you want to look over those too. I think there needs to be a little more clarification on this text: “but generally indicate that organismal responses to starvation and heat tolerance are closely related”

Does this mean that starvation and heat shock have the same effects on survival? Or that starvation has an effect on heat tolerance? If the later, we should try to be specific about the effect it has: Does it tend to increase or decrease thermal limits?